ce/artifacts/activation.ts

```diff
@@ -2,13 +2,67 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-import { delimiter } from 'path';
+import { delimiter, extname } from 'path';
+import { undo as undoVariableName } from '../constants';
+import { i } from '../i18n';
 import { Session } from '../session';
-import { linq } from '../util/linq';
+import { Dictionary, linq } from '../util/linq';
 import { Uri } from '../util/uri';
 import { toXml } from '../util/xml';
 import { Artifact } from './artifact';
 
+
+function generateCmdScript(variables: Dictionary<string>, aliases: Dictionary<string>): string {
```

Comment:
I asked for a second opinion on what this is doing with other vcpkg maintainers who have more bash experience.

Reply:
these look good to me, but I'm probably the least experienced with bash :P